core/roboconf-core/src/test/java/net/roboconf/core/model/RecipesValidatorTest.java

```diff
@@ -47,6 +50,25 @@
 	@Rule
 	public TemporaryFolder folder = new TemporaryFolder();
 
+	private final Logger logger = Logger.getLogger(this.getClass().getName());
+
+	/**
+	 * @return {@code true} if and only if puppet is installed.
+	 * @throws IOException when something bad happened.
+	 */
+	private boolean puppetIsInstalled() throws IOException {
+		int exitCode;
+		try {
+			exitCode = ProgramUtils.executeCommand(
+					this.logger,
+					new String[]{"which", "puppet"},
```

Comment:
Use **puppet -v** instead, as it is more portable among operating systems (yes, including Windows and Mac).